### PR TITLE
Issue 2

### PR DESCRIPTION
**Pull request by chibenwa**
Work in progress. I still need to add automated creation for this workflow script.

This is a rework of a previous pull request.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -45,3 +50,43 @@ Then, you can open it in your browser :
...
+with the James TLS certificates. You may not want to publish a container with 
+such sensitive content, or even commit it in a source code management system.
+
+First copy James TLS certificate to the directory containing this README. Or you can generate them using :
+
+----
+$keytool -genkey -alias james -keyalg RSA -keystore keystore
+----
+
+NOTE: To generate James keystore this way, you need to have keytool installed.
```
Comment (mbaechler) [inline README.adoc]: where does it come from ? (answer : jdk)

File: README.adoc [hunk 2]
```diff
@@ -45,3 +50,43 @@ Then, you can open it in your browser :
...
+
+Be sure that the password you provide matches with James configuration. You will 
+find it in james-parent project, in the destination/conf directory. Concerned files are
```
Comment (mbaechler) [inline README.adoc]: concern doesn't mean what you thing

File: plugins.txt [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+workflow-durable-task-step:1.8
+workflow-api:1.8
+workflow-support:1.8
+workflow-cps:1.8
+workflow-cps-global-lib:1.8
+workflow-job:1.8
+workflow-basic-steps:1.8
+workflow-aggregator:1.8
+stash-pullrequest-builder:1.3.1
+ssh-agent:1.7
```
Comment (mbaechler) [inline plugins.txt]: alphabetic ordering ?

**Comment by aduprat** [comment]
Replaced by:
https://github.com/linagora/james-jenkins/pull/4

**Pull request closed without merging**